### PR TITLE
Implement override plural forms

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1184708

### CODE REVIEW

**Comment by huangp** [comment]
:+1:

**Comment by huangp** [comment]
:+1:

**Comment by djansen-redhat** [comment]
- [ ] :bug: First view of a language does not select the Members tab

File: functional-test/src/test/java/org/zanata/feature/language/AddLanguageTest.java [hunk 1]
```diff
@@ -106,84 +103,84 @@ public void addLanguageAsEnabled() throws Exception {
...
-        assertThat(languageInfo.get("Native Name"))
-                .isEqualTo("русский (Россия)")
-                .as("The native name is correct");
-        assertThat(languageInfo.get("Language Code"))
-                .isEqualTo("ru")
-                .as("The language is correct");
-        assertThat(languageInfo.get("Country Code"))
-                .isEqualTo("RU")
-                .as("The country code is correct");
-    }
```
Comment (djansen-redhat) [inline functional-test/src/test/java/org/zanata/feature/language/AddLanguageTest.java]: Why are these two tests commented out?
If there's a serious problem with them, then @ Ignore them. If they are no longer valid (I think they still are?) then remove them entirely.

**Comment by djansen-redhat** [comment]
@zanata-jenkins retest this please

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/062565c69a67bdfba201613c3a47a41a012a0edf

**Pull request closed without merging**